variables.tf

```diff
@@ -167,3 +167,9 @@ variable "disable_api_termination" {
   description = "If true, enables EC2 Instance Termination Protection"
   default     = "false"
 }
+
+variable "enable_ssm_access" {
+  type        = string
+  description = "If true, attaches SSM policy to instance role"
+  default     = "false"
```

Comment:
@fgauchat-binbash Please change the type to `bool` and set the default value to `false` (without the double quotes).